base/applications/rapps/include/gui.h

```diff
@@ -57,12 +57,14 @@ class CMainWindow : public CWindowImpl<CMainWindow, CWindow, CFrameWinTraits>
     CAtlList<CAppInfo *> m_Selected;
 
     BOOL bUpdating = FALSE;
+    BOOL bAppwizMode;
+    HTREEITEM hTreeInstalled;
 
     CStringW szSearchPattern;
     AppsCategories SelectedEnumType;
 
   public:
-    CMainWindow(CAppDB *db);
+    CMainWindow(CAppDB *db, BOOL bAppwiz = FALSE);
```

Comment:
nit: I am not a fan of default arguments for constructors.

Reply:
If you decide to keep it, then this ctor should be marked `explicit` (yes, that needed to be there before as well)

Reply:
Addressed with the `explicit`